client/ayon_deadline/plugins/publish/maya/submit_maya_deadline.py

```diff
@@ -599,7 +602,8 @@ def smart_replace(string, key_values):
             {"<Scene>": scene_filename_no_ext,
              "<Layer>": layer})
 
-        start_frame = int(self._instance.data["frameStartHandle"])
+
```

Comment:
```suggestion
```

This PR seems to pass the linter check - @antirotor should we make `ayon-deadline` more strict? You seem to be the master of linters?

Reply:
Made PR https://github.com/ynput/ayon-deadline/pull/146